src/automations/checkInactivity.js

```diff
@@ -66,15 +73,9 @@ async function checkInactivePullRequest(pullRequest) {
   const repoOwner = pullRequest.base.repo.owner.login;
   const number = pullRequest.number;
 
-  const labels = await this.issues.getIssueLabels({
-    owner: repoOwner, repo: repoName, number: number
-  });
-  const inactiveLabel = labels.data.find(l => {
-    return l.name === this.cfg.activity.inactive;
-  });
-  const reviewedLabel = labels.data.find(l => {
-    return l.name === this.cfg.activity.pullRequests.reviewed.label;
-  });
+  const labels = await extractPRLabels.call(this, repoOwner, repoName, number);
```

Comment:
Instead of calling the function again and making 2 API requests, can't you just pass the `label` object to the `checkInactivePullRequest()` function as a parameter?